.github/ISSUE_TEMPLATE/suggest-a-new-rule.md

```diff
@@ -0,0 +1,20 @@
+---
+name: New rule suggestion
+about: Suggest a new rule
+title: '[New rule] '
+labels: ''
+assignees: ''
+
+---
+
+## Describe the problem you are trying to solve
```

Comment:
💡 I prefer `###`. It's big enough to stand out even when the issue has larger code blocks, but not so big as to be distracting.